build/yaml/templates/component-template.yml

```diff
@@ -15,6 +15,7 @@ stages:
         - job: Build
           displayName: Build and package
           steps:
+          - template: versioning-steps.yml
           - template: dotnet-package-steps.yml
```

Comment:
I was expecting to see some kind of corresponding change to dotnet-package-steps.yml similar to how you've done for npm and nuspec. Is it not needed?

Reply:
Nope! MSBuild picks up the $(VersionSuffix) environment variable on its own. I thought that I'd need to explicitly include it in packing steps but it worked automatically once the variable was configured.

Reply:
Ahh, wonderful! Kinda magic but works for me!